Lib/pdb.py

```diff
@@ -1132,9 +1132,9 @@ def do_args(self, arg):
         """
         co = self.curframe.f_code
         dict = self.curframe_locals
-        n = co.co_argcount
-        if co.co_flags & 4: n = n+1
-        if co.co_flags & 8: n = n+1
+        n = co.co_argcount + co.co_kwonlyargcount
```

Comment:
You also need to add co.co_posonlyargcount to handle PEP570 (check the issue). Although this will difficult backporting, so maybe you want this to be done in a separate PR.

Reply:
Thank @pablogsal, it may be simple to do so in a separate PR I think.